mars/deploy/local/session.py

```diff
@@ -129,7 +129,7 @@ def run(self, *tileables, **kw):
         exec_start_time = time.time()
         time_elapsed = 0
         while timeout <= 0 or time_elapsed < timeout:
-            timeout_val = min(5, timeout - time_elapsed) if timeout > 0 else 5
+            timeout_val = min(20, timeout - time_elapsed) if timeout > 0 else 20
```

Comment:
How about define a constant value about the timeout? I remember there is not just one place to use it.